setup.ts

```diff
@@ -246,6 +247,7 @@ export async function checkDb(url: string): Promise<boolean> {
     } else {
       dbEmpty = true;
     }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Remove the linting disable. Find an acceptable solution. We have a PR test to make sure they are not used.